### ISSUE TITLE
Beheading freezes the game

### ISSUE THREAD

**Issue by Insane96** [PrinceOfAmber/Cyclic#1912]
Minecraft Version: 1.16.5

Forge Version: 36.2.0

Mod Version: 1.3.4

Single Player or Server: Server

Describe problem (what you were doing; what happened; what should have happened):
When a mob drops an head with beheading (only heads that aren't already in game) the game freezes.
This is due the head begin loaded from the minecraft servers in the time the game freezes.

Log file link:
Not needed

Video/images/gifs (direct upload or link): 

https://user-images.githubusercontent.com/7185104/132943274-c0f0ff8c-53fa-4029-bc61-a2f4b929f2e1.mp4

**Comment by mc-kaishixiaxue** [comment]
Try to `set beheadingEntityMHF = []` on cyclic.toml, it makes this enchantment no longer drop the player's head.
However, it's treating the symptom but not the root cause. Dropping player's head by beheading is a default setting. After deleting the config file to reset settings, the issue will be still reproduced.

**Comment by Lothrazar** [comment]
I think this is just how the Player head blocks work by default.  Above solution will work in config.

I can add a new true/false config somewhere if that would help, but then it would just drop unskinned heads

**Comment by Insane96** [comment]
What about using something like the Mob Heads mod?

Il Mer 15 Set 2021, 04:16 Sam ***@***.***> ha scritto:

**Comment by Lothrazar** [comment]
Yeah i thought about relying on some other mob heads mod, but if i add optional dependency to a mod , then just like with curios and JEI i would want that mod to be updated to modern versions and have consistent support. for example with this https://www.curseforge.com/minecraft/mc-mods/mob-heads , that would support 1.16.5 but not my newer branches. so all that work could not be reused in 1.18.2, 1.19, 1.20, etc... .  i will re look at this as an enhancement in modern versions, relabelling from bug to enhancement

**Comment by Insane96** [comment]
Yeah I abandoned that mod since wasn't really used, and alternatives like Vanilla Tweaks exist
